### ISSUE TITLE
Keep getting "Invalid Source"

### ISSUE THREAD

**Issue by SessamiApp** [exelix11/SysDVR#121]
Hi, possibly an easy fix?

I keep getting:
_"SysDVR-Client - 5.0.2 by exelix
https://github.com/exelix11/SysDVR 

Initializing video player with h264 codec.
Invalid source"_ 

Running Terminal on MacOS Big Sur.
Re-installed drivers, tried every mode on the Switch SysDVR settings.

VLC doesn't seem to open the rtsp://<switch ip> either so maybe it's a switch issue but I have everything installed correctly. 
Even ran the Archive Bit fix in the hopes it was that.

Tried SysDVR 5.0 (same version on switch and SysDVR-Client) but that did the same thing. 

Pulling my hair out here and really want to get this working for an online family get together during lockdown.
Thank in advance for any suggestions!

**Comment by exelix11** [comment]
>  Invalid source

What is the command you're trying to run ?

**Comment by SessamiApp** [comment]
"dotnet SysDVR-Client.dll SysDVR-Client usb"

**Comment by exelix11** [comment]
it's `dotnet SysDVR-Client.dll usb` you are typing the program name twice.

**Comment by SessamiApp** [comment]
I did think that but arrived at that as that was the closest I got to seeing anything happen. If I put "dotnet SysDVR-Client usb"

I get:
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-SysDVR-Client does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
  
  P.S. A little bit starstruck dealing with the creator of this...top work matey!
